src/main/js/distribution/ProjectConfiguration.js

```diff
@@ -3,13 +3,19 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
 import { getProjects } from 'store/actions/projects';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 function assignClassName(row, rowIdx) {
     return 'tableRow';
 }
 
 function assignDataFormat(cell, row) {
-    const cellContent = (row.missing) ? <span className="missingBlackDuckData"><span className="fa fa-exclamation-triangle fa-fw" aria-hidden="true" />{cell}</span> : cell;
+    const cellContent = (row.missing) ? <span className="missingBlackDuckData">
+        <span className="fa-layers fa-fw">
+            <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />{cell}
+        </span> </span>
+        : cell;
+
```

Comment:
Can we format this differently? It's a bit hard to read at the moment. I'd recommend:
`const cellContent = (row.missing) ?
    <span className="missingBlackDuckData">
        <span className="fa-layers fa-fw">
            <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />{cell}
        </span>
     </span>
        : cell;`

Or something similar. It's a very large ternary operator, so even a variable for the HTML might be a better option.

Reply:
That did not format at all. So I think another variable for the HTML might make this easier to read.